quarkus-reactive-kafka-consumer/pom.xml

```diff
@@ -51,17 +54,32 @@
       <artifactId>rest-assured</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-kafka-client</artifactId>
+    </dependency>
     <dependency>
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-smallrye-reactive-messaging-kafka</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-apicurio-registry-avro</artifactId>
+    </dependency>
+    <!--
+    <dependency>
+      <groupId>io.apicurio</groupId>
+      <artifactId>apicurio-registry-utils-serde</artifactId>
+      <version>${apicurio.version}</version>
+    </dependency>
+    -->
```

Comment:
One example project I found required `apicurio-registry-utils-serde for serialization/deserialization. The `apicurio.version` was incompatible with this dependency so I commented this code out. It may be needed once we start reading the events in the consumer code.